webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -54,21 +54,53 @@ class SakaiRubricsManager extends RubricsElement {
           </div>
         </div>
 
-      <div role="tabpanel" aria-labelledby="site-rubrics-title" id="site_rubrics">
-        <div class="rubric-title-sorting">
-          <div @click="${this.sortRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site-name" key="site_name">site_name</sr-lang><span class="collpase-icon fa fa-chevron-up site-name sort-element-site"></span></a></div>
-          <div @click="${this.sortRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site-title" key="site_title">site_title</sr-lang><span class="collpase-icon fa site-title sort-element-site"></span></a></div>
-          <div @click="${this.sortRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site-creator" key="creator_name">creator_name</sr-lang><span class="collpase-icon fa site-creator sort-element-site"></span></a></div>
-          <div @click="${this.sortRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site-modified" key="modified">modified</sr-lang><span class="collpase-icon fa site-modified sort-element-site"></span></a></div>
-          <div class="actions"><sr-lang key="actions">actions</sr-lang></div>
+        <div role="tabpanel" aria-labelledby="site-rubrics-title" id="site_rubrics">
+          <div class="rubric-title-sorting">
+            <div>
+              <a href="javascript:void(0)"
+                  @click=${this.sortRubrics}
+                  style="text-decoration: none;"
```

Comment:
I think the `<a>` tag already has the default CSS of text-decoration set to none